<?xml version="1.0"?>

<launch>

	<!-- Launch du minimal bringup et des sons perso-->
	<include file="$(find turtlebot_bringup)/launch/minimal.launch">
	</include>
	<include file="$(find sound_publisher)/launch/sound_publisher.launch">
	</include>

	<!-- Launch facedetection and depth analysis-->
	<include file="$(find camera_detection)/launch/camera_detection.launch">
	</include>

	<!-- Lancement des noeuds du scenario -->
	<!--    - Orientation de la camÃ©ra -->
	<node pkg='turtlebot_scenario' type='servo_motion.py' name='servo_motion' output='screen'>
	</node>

	<node pkg='turtlebot_scenario' type='led_control.py' name='led_control' output='screen'/>

	<!--    - Lancement du scenario -->
	<node pkg='turtlebot_scenario' type='scenario.py'  name='turtlebot_scenario2' output='screen'>
	</node>
	
</launch>
